packages/react-native-reanimated/Common/cpp/reanimated/NativeModules/NativeReanimatedModule.h

```diff
@@ -168,18 +171,23 @@ class NativeReanimatedModule : public NativeReanimatedModuleSpec {
     return *layoutAnimationsManager_;
   }
 
-  inline jsi::Runtime &getUIRuntime() const {
+  [[nodiscard]] inline jsi::Runtime &getUIRuntime() const {
     return uiWorkletRuntime_->getJSIRuntime();
   }
 
-  inline bool isBridgeless() const {
+  [[nodiscard]] inline bool isBridgeless() const {
```

Comment:
why do we need `[[nodiscard]]` for those functions? 🤔

Reply:
https://github.com/software-mansion/react-native-reanimated/pull/6539#discussion_r1776755872